pyproject.toml

```diff
@@ -32,11 +32,12 @@ dependencies = [
     "defusedxml~=0.7",
     "imageio-ffmpeg~=0.6",
     "joblib~=1.5",
+    "notebook>=7.4.4",
```

Comment:
This shouldn't be a required dependency. To run notebooks, we already have the extras `notebooks`

Reply:
Removed, I think this got added by pycharm when I clicked install missing dependencies.